src/index.html

```diff
@@ -1,101 +1,100 @@
+        <article>
+            <h2 class='category-four'>category four</h2>
+        </article>
+        <article id= 'clue' class='col column-one-100' > 
+            <h3>100</h3>
+        </article>
+        <article id= 'clue' class='col column-two-100'> 
+            <h3>100</h3>
+        </article>
+        <article id= 'clue' class='col column-three-100'>
             <h3>100</h3>
         </article>
-        <article id= 'clue' class='100'> 
+        <article id= 'clue' class='col column-four-100'> 
             <h3>100</h3>
         </article>
-        <article id= 'clue' class='200'>
+        <article id= 'clue' class='col column-one-200'>
             <h3>200</h3>
         </article>
-        <article id= 'clue' class='200'>
+        <article id= 'clue' class='col column-two-200'>
             <h3>200</h3>
         </article>
-        <article id= 'clue' class='200'>
+        <article id= 'clue' class='col column-three-200'>
             <h3>200</h3>
         </article>
-        <article id= 'clue' class='200'>
+        <article id= 'clue' class='col column-four-200'>
             <h3>200</h3>
         </article>
-        <article id= 'clue' class='300'>
+        <article id= 'clue' class='col column-one-300'>
             <h3>300</h3>
         </article>
-        <article id= 'clue' class='300'>
+        <article id= 'clue' class='col column-two-300'>
             <h3>300</h3>
         </article>
-        <article id= 'clue' class='300'>
+        <article id= 'clue' class='col column-three-300'>
             <h3>300</h3>
         </article>
-        <article id= 'clue' class='300'>
+        <article id= 'clue' class='col column-four-300'>
             <h3>300</h3>
         </article>
-        <article id= 'clue' class='400'>
+        <article id= 'clue' class='col column-one-400'>
             <h3>400</h3>
         </article>
-        <article id= 'clue' class='400'>
+        <article id= 'clue' class='col column-two-400'>
             <h3>400</h3>
         </article>
-        <article id= 'clue' class='400'>
+        <article id= 'clue' class='col column-three-400'>
             <h3>400</h3>
         </article>
-        <article id= 'clue' class='400'>
+        <article id= 'clue' class='col column-four-400'>
             <h3>400</h3>
         </article>
       </section>
     </main>
     <section class='round'>
       <h4> Round <span class='round-number'> 1</span>
-      </h3>
+      </h4>
     </section>
     <section class='players'>
-      <div id='players' class='player-one'>
+        <form id='players' class='player-one'>
             <label class='player-one-name'>Player One</label><br/>
             <input type='text' class='player-one-input' value= 'Player One'/>
             <h5>Score</h5>
             <h6>0</h6>
-        </div>
-        <div id='players' class='player-two'>
+        </form>
+        <form id='players' class='player-two'>
             <label class='player-two-name'>Player Two</label><br/>
```

Comment:
This would be a good opportunity to link labels and inputs by using `ARIA` properties. So something like this: 

```js
<label class='player-one-name' for='player-one-input'>Player One</label><br/>
<input type='text' id='player-one-input'/>
```